docs/documentation/models.rst

```diff
@@ -325,3 +325,15 @@ Wind chill temperature
 .. autoclass:: pythermalcomfort.classes_return.WCT
     :members:
 
+Work capacity
+---------
+
+.. autofunction:: pythermalcomfort.models.work_capacity.workcapacity_dunne
+
+.. autofunction:: pythermalcomfort.models.work_capacity.workcapacity_hothaps
+
+.. autofunction:: pythermalcomfort.models.work_capacity.workcapacity_iso
+
+.. autofunction:: pythermalcomfort.models.work_capacity.workcapacity_niosh
+.. autoclass:: pythermalcomfort.classes_return.WorkCapacity
```

Comment:
maybe move this at the top if it is shared across all models, right under the description of the work capacity models